tests/cpgrid/addLgrsOnDistributedGrid_test.cpp

```diff
@@ -542,6 +576,19 @@ BOOST_AUTO_TEST_CASE(atLeastOneLgr_per_process_attempt)
         // Total global ids in leaf grid view for cells: 36-(6 marked cells) + 16 + 27 + 64 + 16 = 153
         // Total global ids in leaf grid view for points: 80 + 33 + 56 + 117 + 33 = 319
         refinePatch_and_check(grid, cells_per_dim_vec, startIJK_vec, endIJK_vec, lgr_name_vec);
+
+        // Check global id is not duplicated for points
+        std::vector<int> localPointIds_vec;
+        localPointIds_vec.reserve(grid.currentData().back()->size(3));
+        for (const auto& point : vertices(grid.leafGridView())) {
+            // Notice that all partition type points are pushed back. Selecting only interior points does not bring us to the expected value.
+            localPointIds_vec.push_back(grid.currentData().back()->globalIdSet().id(point));
+        }
+        auto [allGlobalIds_points, displPoint ] = Opm::allGatherv(localPointIds_vec, grid.comm());
+        const std::set<int> allGlobalIds_points_set(allGlobalIds_points.begin(), allGlobalIds_points.end());
+
+        // Total global ids in leaf grid view for points: 80 + 33 + 56 + 117 + 33 = 319
+        BOOST_CHECK( allGlobalIds_points_set.size() == 319 );
     }
```

Comment:
Same here: Next step?

Reply:
See my comment above, please. It's another test of fully interior LGRs. I would like to keep it as well :)